### PR TITLE
lvm operator: add initial CI config for e2e test

### PR DESCRIPTION
**Pull request by riya-singhal31**
Added the CI config for e2e tests of lvm operator.

Signed-off-by: riya-singhal31 <riyasinghalji@gmail.com>

### CODE REVIEW

File: ci-operator/jobs/red-hat-storage/lvm-operator/red-hat-storage-lvm-operator-main-presubmits.yaml [hunk 1]
```diff
@@ -49,3 +49,77 @@ presubmits:
...
+    context: ci/prow/e2e-test
+    decorate: true
+    decoration_config:
+      skip_cloning: true
+    labels:
+      ci-operator.openshift.io/cloud: aws
+      ci-operator.openshift.io/cloud-cluster-profile: aws
+      ci.openshift.io/generator: prowgen
+      pj-rehearse.openshift.io/can-be-rehearsed: "true"
+    name: pull-ci-red-hat-storage-odf-multicluster-orchestrator-main-e2e-test
```
Comment (sp98) [inline ci-operator/jobs/red-hat-storage/lvm-operator/red-hat-storage-lvm-operator-main-presubmits.yaml:67]: This name needs to be fixed to used lvm operator.

Comment (riya-singhal31) [reply ci-operator/jobs/red-hat-storage/lvm-operator/red-hat-storage-lvm-operator-main-presubmits.yaml]: Updated

File: ci-operator/config/red-hat-storage/lvm-operator/red-hat-storage-lvm-operator-release-4.10.yaml [hunk 2]
```diff
@@ -31,6 +31,10 @@ tests:
   commands: make test
   container:
     from: test-bin
+- as: e2e-test
```
Comment (nbalacha) [inline ci-operator/config/red-hat-storage/lvm-operator/red-hat-storage-lvm-operator-release-4.10.yaml:34]: Not required. The e2e tests are not available in the release-4.10 branch.

File: ci-operator/jobs/red-hat-storage/lvm-operator/red-hat-storage-lvm-operator-main-presubmits.yaml [hunk 3]
```diff
@@ -49,3 +49,53 @@ presubmits:
...
+      skip_cloning: true
+    labels:
+      ci.openshift.io/generator: prowgen
+      pj-rehearse.openshift.io/can-be-rehearsed: "true"
+    name: pull-ci-red-hat-storage-lvm-operator-main-e2e-test
+    rerun_command: /test e2e-test
+    spec:
+      containers:
+      - args:
+        - --gcs-upload-secret=/secrets/gcs/service-account.json
```
Comment (nbalacha) [inline ci-operator/jobs/red-hat-storage/lvm-operator/red-hat-storage-lvm-operator-main-presubmits.yaml:70]: Does the Openshift-CI use aws or GCS? I thought it was AWS.

File: ci-operator/jobs/red-hat-storage/lvm-operator/red-hat-storage-lvm-operator-release-4.10-presubmits.yaml [hunk 4]
```diff
@@ -49,3 +49,52 @@ presubmits:
         secret:
           secretName: result-aggregator
     trigger: (?m)^/test( | .* )unit-test,?($|\s.*)
+  - agent: kubernetes
```
Comment (nbalacha) [inline ci-operator/jobs/red-hat-storage/lvm-operator/red-hat-storage-lvm-operator-release-4.10-presubmits.yaml:52]: Not required. e2e-tests are not available in the release-4.10 branch.

**Review by nbalacha — CHANGES_REQUESTED** [review]

File: ci-operator/config/red-hat-storage/lvm-operator/red-hat-storage-lvm-operator-release-4.10.yaml (hunk 2 above)
Comment (riya-singhal31) [reply ci-operator/config/red-hat-storage/lvm-operator/red-hat-storage-lvm-operator-release-4.10.yaml]: Updated

File: ci-operator/jobs/red-hat-storage/lvm-operator/red-hat-storage-lvm-operator-release-4.10-presubmits.yaml (hunk 4 above)
Comment (riya-singhal31) [reply ci-operator/jobs/red-hat-storage/lvm-operator/red-hat-storage-lvm-operator-release-4.10-presubmits.yaml]: Updated

File: ci-operator/jobs/red-hat-storage/lvm-operator/red-hat-storage-lvm-operator-release-4.10-presubmits.yaml [hunk 5]
```diff
@@ -49,3 +49,4 @@ presubmits:
         secret:
           secretName: result-aggregator
     trigger: (?m)^/test( | .* )unit-test,?($|\s.*)
+
```
Comment (sp98) [inline ci-operator/jobs/red-hat-storage/lvm-operator/red-hat-storage-lvm-operator-release-4.10-presubmits.yaml:52]: remove these blank links at the end.

File: ci-operator/jobs/red-hat-storage/lvm-operator/red-hat-storage-lvm-operator-main-presubmits.yaml [hunk 6]
```diff
@@ -49,3 +49,53 @@ presubmits:
...
+      serviceAccountName: ci-operator
+      volumes:
+      - name: pull-secret
+        secret:
+          secretName: registry-pull-credentials
+      - name: result-aggregator
+        secret:
+          secretName: result-aggregator
+    trigger: (?m)^/test( | .* )e2e-test,?($|\s.*)
+
```
Comment (sp98) [inline ci-operator/jobs/red-hat-storage/lvm-operator/red-hat-storage-lvm-operator-main-presubmits.yaml:101]: remove these blank links at the end.

**Review by sp98 — CHANGES_REQUESTED** [review]

File: ci-operator/jobs/red-hat-storage/lvm-operator/red-hat-storage-lvm-operator-release-4.10-presubmits.yaml (hunk 5 above)
Comment (riya-singhal31) [reply ci-operator/jobs/red-hat-storage/lvm-operator/red-hat-storage-lvm-operator-release-4.10-presubmits.yaml]: Updated

File: ci-operator/jobs/red-hat-storage/lvm-operator/red-hat-storage-lvm-operator-main-presubmits.yaml (hunk 6 above)
Comment (riya-singhal31) [reply ci-operator/jobs/red-hat-storage/lvm-operator/red-hat-storage-lvm-operator-main-presubmits.yaml]: Done

**Review by agarwal-mudit — APPROVED** [review]
/lgtm

**Pull request merged**